application/ai/knowledge_llm_contract.py

```
@@ -89,13 +98,40 @@ def initial_knowledge_openai_function_tool() -> Dict[str, Any]:
 # ---------------------------------------------------------------------------
 
 
+def parse_json_from_response(rsp: str):
+    """从LLM响应中解析JSON，支持```json包裹格式"""
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)